pkg/kv/kvserver/replica_proposal_buf.go

```diff
@@ -521,24 +635,14 @@ func (b *propBuf) FlushLockedWithRaftGroup(
 				firstErr = err
 				continue
 			}
-		} else {
-			// Add to the batch of entries that will soon be proposed. It is
```

Comment:
reflection: Swapping the order of `if` and `else` clauses here made the diff unnecessarily bigger.